src/popup/components/TransferReview.vue

```diff
@@ -130,7 +131,8 @@ export default {
       'tippingV2',
     ]),
     isRecipientName() {
-      return this?.recipientAddress && checkAensName(this.recipientAddress);
+      return this.transferData?.address
```

Comment:
I suggest to make the multiline conditions in a way that every item is in separate line.
This way:
```
return (
  condition
  && condition
)
```